plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3select/TestS3SelectPushdown.java

```diff
@@ -276,17 +276,17 @@ public void testShouldEnableSplits()
         assertTrue(isSplittable(true, schema, inputFormat, new Path("s3://fakeBucket/fakeObject.csv")));
         // Pushdown disabled
         assertTrue(isSplittable(false, schema, inputFormat, new Path("s3://fakeBucket/fakeObject.csv")));
+        // JSON
```

Comment:
`JSON` -> `Uncompressed JSON`

Reply:
Agreed. My comment was thought to be in sync with

https://github.com/trinodb/trino/pull/14040/files#diff-e49429ea9b31f6935ffb8fceb7241e4a6554ebb3ac1c103a8d2129180a56da74R275